python_cli_skeleton_app/app_config.py

```diff
@@ -14,16 +14,14 @@ class AppConfig:
 
     # unfortunately, this class needs to hard-codedly know the location of the application configuration file
     root_dir = Path(__file__).parent.parent
-    config.read(f"{root_dir}/{app_config_filename}")
+    config.read(f"{root_dir}/{app_config_filename}")  # noqa: CCE001, CCE002
 
     app_env = os.getenv("APP_ENV")
-    if app_env is None:
+    if app_env is None:  # noqa: CCE002
```

Comment:
Ignoring rules as this `app_config` class is meant to be globally available and have class-level expresssions.